@xen-orchestra/web/src/components/storage-repositories/StorageRepositoriesTable.vue

```diff
@@ -0,0 +1,193 @@
+import type { IconName } from '@core/icons'
+import VtsDataTable from '@core/components/data-table/VtsDataTable.vue'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import VtsStateHero from '@core/components/state-hero/VtsStateHero.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCheckbox from '@core/components/ui/checkbox/UiCheckbox.vue'
+import UiLink from '@core/components/ui/link/UiLink.vue'
+import UiQuerySearchBar from '@core/components/ui/query-search-bar/UiQuerySearchBar.vue'
+import UiTablePagination from '@core/components/ui/table-pagination/UiTablePagination.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import UiTopBottomTable from '@core/components/ui/top-bottom-table/UiTopBottomTable.vue'
+import { usePagination } from '@core/composables/pagination.composable.ts'
+import { useRouteQuery } from '@core/composables/route-query.composable.ts'
+import { useTable } from '@core/composables/table.composable.ts'
+import { vTooltip } from '@core/directives/tooltip.directive.ts'
+import { noop } from '@vueuse/shared'
+import { computed, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { srs } = defineProps<{
+  srs: XoSr[]
+  hasError: boolean
+}>()
+
+const { t } = useI18n()
+
+const selectedSrId = useRouteQuery('id')
+
+const searchQuery = ref('')
+
+const filteredSrs = computed(() => {
+  const searchTerm = searchQuery.value.trim().toLocaleLowerCase()
+
+  if (!searchTerm) {
+    return srs
+  }
+
+  return srs.filter(sr => Object.values(sr).some(value => String(value).toLocaleLowerCase().includes(searchTerm)))
+})
+
+const { visibleColumns, rows } = useTable('backup-jobs', filteredSrs, {
+  rowId: record => record.id,
+  columns: define => [
+    define('checkbox', noop, { label: '', isHideable: false }),
+    define('name', record => ({ name: record.name_label, id: record.id }), { label: t('storage-repository') }),
+    define('description', record => record.name_description, { label: t('description') }),
+    define('storage-format', record => record.SR_type, { label: t('storage-format') }),
+    define('shared', record => record.shared, { label: t('shared') }),
+    define('size', record => ({ used: record.physical_usage, total: record.size }), { label: t('size') }),
+    // define('mode', record => getModeLabels(record), { label: t('mode') }),
+    // define('last-runs', record => getLastThreeRunsStatuses(record), {
+    //   label: t('last-n-runs', { n: 3 }),
+    // }),
+    define('more', noop, { label: '', isHideable: false }),
+  ],
+})
+
+const { pageRecords: srsRecords, paginationBindings } = usePagination('srs', rows)
+
+type BackupJobHeader = 'name' | 'description' | 'storage-format' | 'shared' | 'size'
+
+const headerIcon: Record<BackupJobHeader, IconName> = {
+  name: 'fa:align-left',
+  description: 'fa:align-left',
+  'storage-format': 'fa:square-caret-down',
+  shared: 'fa:square-caret-down',
+  size: 'fa:hashtag',
```

Comment:
```suggestion
  space: 'fa:a',
```
This icon was added in a [PR](https://github.com/vatesfr/xen-orchestra/pull/9018) of Enée (not merge yet , I think)